src/applications/disability-benefits/all-claims/components/ComboBox.jsx

```diff
@@ -322,6 +346,7 @@ export class ComboBox extends React.Component {
           onKeyDown={this.handleKeyPress}
           ref={this.inputRef}
           message-aria-describedby={autocompleteHelperText}
+          role="combobox"
```

Comment:
Adding this role changes how screenreaders interpret this component and as a result the listbox of matching results isn't reachable via screenreader navigation. If you want to try it out, you can on mac/voiceover with CTRL + OPTION + left or right arrows.  Let me know if you want me to show you or record a vid. We can remove this.

Reply:
I will say it's definitely counterintuitive that our `Combobox` component shouldn't have the "combobox" role 🫤 . It could make more sense to add later when we rejigger the experience, but for now we definitely want to keep the experience predictable for veterans. Thanks for updating!